zoom_operation_rules/zoom_operation_user_granted_admin.py

```diff
@@ -0,0 +1,18 @@
+from panther_zoom_helpers import get_zoom_user_context as get_context
```

Comment:
I did not use the Standard admin access rule here because the Zoom data model does not contain all the fields that rule requires